test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,32 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind an imported Node.js application to Etcd database
+        Given Etcd operator running
+        Given Etcd cluster "etcd-cluster-example" is created
+        * Imported Nodejs application "nodejs-rest-http-crud-etcd" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+              name: binding-request-etcd
+            spec:
+              applicationSelector:
+                group: apps
+                version: v1
+                resource: deployments
+                resourceRef: nodejs-rest-http-crud-etcd
+              backingServiceSelector:
+                group: etcd.database.coreos.com
+                version: v1beta2
+                kind: EtcdCluster
+                resourceRef: etcd-cluster-example
+              detectBindingResources: true
+            """
+        Then deployment must contain intermediate secret
+            | intermediate_secret_name |
+            | binding-request-etcd     |
```

Comment:
from user perspective, this in unimportant - instead user wants that NodeJS application works. We should check that instead.

Reply:
added this because of [https://github.com/redhat-developer/service-binding-operator/pull/567#discussion_r472015323](url)

Can add this step too, `Then application should be connected to the DB "etcd-cluster-example"`, but then we have to generalize this step as `Then application should be connected tot the backing service instance`. your thoughts on this.

Reply:
if you can check that the application is working properly, then you do not need this step here - because implicitly we are going to check that. We are not really interested in binding secrets - we are interested that bindings are injected into application so that application works.

Reply:
This commit fixes the use of nodejs todo app instead of nodejs fruit app. 
https://github.com/shvasude/service-binding-operator/commit/e0d93ef37eeb73a61859e5e6220a063a7d4ea67c